packages/addon-dev/src/rollup-glint-plugin.ts

```diff
@@ -0,0 +1,25 @@
+import type { Plugin } from 'rollup';
+import { execaCommand } from 'execa';
+import { fixBadDeclarationOutput } from 'fix-bad-declaration-output';
+
+export default function rollupGlintPlugin(pattern: string): Plugin {
+  return {
+    name: 'rollup-glint-plugin',
+
+    async closeBundle() {
+      /**
+       * Generate the types (these include /// <reference types="ember-source/types"
+       * but our consumers may not be using those, or have a new enough ember-source that provides them.
+       */
+      await execaCommand(`pnpm glint --declaration`, { stdio: 'inherit' });
```

Comment:
Why `pnpm`? execa should be able to find the `glint` executable using `preferLocal: true`

Reply:
thanks!